Cargo.toml

```diff
@@ -16,8 +16,8 @@ resolver = "2"
 unexpected_cfgs = { level = "warn", check-cfg = ['cfg(tokio_unstable)'] }
 
 [workspace.package]
-edition = "2021"
-rust-version = "1.77"
+edition = "2024"
+rust-version = "1.85"
```

Comment:
The  MSRV bump is more important bit for the users so I think that's what the commit shortlog should mention and edition bump could go in the details section.